src/screens/Analytics/Logs/RefundLogs/RefundLogs.res

```diff
@@ -1,151 +1,19 @@
-@module("js-sha256") external sha256: string => string = "sha256"
-
 @react.component
 let make = (~refundId, ~paymentId) => {
   open APIUtils
-  open LogicUtils
-  open PaymentLogsUtils
-  open LogTypes
   let fetchDetails = useGetMethod(~showErrorToast=false, ())
-  let (data, setData) = React.useState(_ => [])
-  let (logDetails, setLogDetails) = React.useState(_ => {
-    response: "",
-    request: "",
-  })
-  let (selectedOption, setSelectedOption) = React.useState(_ => {
-    value: 0,
-    optionType: API_EVENTS,
-  })
-  let isError = React.useMemo0(() => {ref(false)})
-  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
 
   let refundsLogsUrl = `${HSwitchGlobalVars.hyperSwitchApiPrefix}/analytics/v1/api_event_logs?type=Refund&payment_id=${paymentId}&refund_id=${refundId}`
   let webhooksLogsUrl = `${HSwitchGlobalVars.hyperSwitchApiPrefix}/analytics/v1/outgoing_webhook_event_logs?&payment_id=${paymentId}&refund_id=${refundId}`
   let connectorLogsUrl = `${HSwitchGlobalVars.hyperSwitchApiPrefix}/analytics/v1/connector_event_logs?payment_id=${paymentId}&refund_id=${refundId}`
```

Comment:
Same APIUtils.res

Reply:
Current util file doesn't support passing more than one id in url